packages/kbn-build/src/commands/run.js

```diff
@@ -1,6 +1,7 @@
 import chalk from 'chalk';
```

Comment:
__note:__ not related to this PR, but running `kbn run prettier` updates some files in the `kbn-build`:

`packages/kbn-build/src/production/__tests__/fixtures/packages/foo/src/index.js`
`packages/kbn-build/src/production/__tests__/fixtures/packages/bar/src/index.js`

Reply:
Good catch, created https://github.com/elastic/kibana/pull/16668